src/lib/tuning/index/index_evaluator.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <map>
+#include <memory>
+#include <set>
+#include <utility>
+
+#include "sql/sql_query_cache.hpp"
+#include "sql/sql_query_plan.hpp"
+#include "tuning/index/base_index_evaluator.hpp"
+#include "tuning/index/column_ref.hpp"
+#include "tuning/index/index_choice.hpp"
+
+namespace opossum {
+
+/**
+ * This is the default implementation of a BaseIndexEvaluator.
+ *
```

Comment:
Can we find a name for this evaluator? When I saw IndexEvaluator and BaseIndexEvaluator, I expected the first one to be a templated class. Also, our style guide would ask for an AbstractIndexEvaluator (because it is not a non-templated base)

Reply:
renamed `BaseIndexEvaluator` to `AbstractIndexTuningEvaluator`